apps/portal/app-config.yaml

```
@@ -130,24 +141,20 @@ catalog:
-    # - type: url
-    #   target: https://github.com/backstage/backstage/blob/master/packages/catalog-model/examples/acme-corp.yaml
-    #   rules:
-    #     - allow: [User, Group]
+    gitea:
+      helios:
+        host: 127.0.0.1:3030
+        organization: 'helios-platform'
+        catalogPath: 'catalog-info.yaml'
+        branch: 'main'
+        schedule:
+          frequency: { minutes: 5 }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)